pkgs/os-specific/linux/vm-tools/default.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, stdenv, linux }:
+
+stdenv.mkDerivation {
+  pname = "vm-tools";
+  inherit (linux) version src;
+
+  makeFlags = [ "sbindir=${placeholder "out"}/bin" ];
+
+  preConfigure = "cd tools/vm";
```

Comment:
```suggestion
  preConfigure = ''
    cd tools/vm
  '';
```

Reply:
this seems inconsistent with your other suggestions where you prefer a compacter style (i'm referring to the `'' +` suggestions)
also, this change can be made if that line ever has to be multi-line, which seems quite unlikely

Reply:
Commands always use the multiline strings that it is clear that it is a command and there where occasions where the new at the end was missing and caused the command to be not executed.

Reply:
i'm having trouble parsing your sentence,
so i can only respond to what i assume was meant to be "commands always use multiline strings"

i'm finding over 200 occurrences of a single line string for `preConfigure` alone, 2 of which are in the docs